classes/Repository/GanalyticsRepository.php

```diff
@@ -78,6 +93,26 @@ public function addNewRow(array $data, $type = Db::INSERT_IGNORE)
         );
     }
 
+    /**
+     * Adds new order into repository
+     *
+     * @param int $idOrder
+     * @param int $idShop
+     *
+     * @return bool
+     */
+    public function addOrder(int $idOrder, int $idShop)
+    {
+        return $this->addNewRow(
+            [
+                'id_order' => (int) $idOrder,
+                'id_shop' => (int) $idShop,
+                'sent' => 0,
+                'date_add' => ['value' => 'NOW()', 'type' => 'sql'],
```

Comment:
Hoho ... I remember we used to have problems with such things. Possible warning.

@zuk3975 I remember we had a tough time because of something like this when working on a migrated page. Do you remember what it was ? 😅

Reply:
Just moved the code 👍